docs/sdk/access-control/lit-action-conditions.md

```diff
@@ -6,7 +6,7 @@ sidebar_position: 4
 
 You can create a Lit Action Condition to grant access whenever a given Lit Action returns true. Lit Actions are JS code that can be executed on the Lit Protocol network. You can use Lit Actions to create custom access control conditions.
```

Comment:
instead of 'whenever a given Lit Action returns true' what about we say 'whenever a given Lit Action meets the conditions you set'